### DISCUSSION TITLE
Angular Compatibility

### DISCUSSION THREAD

**Discussion by englio15** [nartc/ngxtension-platform#315] [Q&A]
The Compatibility Matrix under the Installation section of the documentation says WIP. Does anyone know what version is compatible with Angular 16?

**Comment by eneajaho** [comment]
@englio15 Most of the utilities depend on v17 features, so I guess that's the version we support.

**Comment by eneajaho** [comment]
More correct table: https://github.com/ngxtension/ngxtension-platform?tab=readme-ov-file#version-compatibility-table
